cli/command/registry_test.go

```diff
@@ -77,3 +138,62 @@ func TestElectAuthServer(t *testing.T) {
 		}
 	}
 }
+
+func TestGetDefaultAuthConfig(t *testing.T) {
+	testCases := []struct {
+		checkCredStore     bool
+		expectErr          bool
+		expectedAuthServer string
+		expectedUsername   string
+		expectedPassword   string
```

Comment:
testify assertions can compare structs, so instead of separate fields for these I think it should be `expected authConfig`, that way you can just `assert.Equal(t, testcase.expected, authconfig)`